pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/queries/QueryRunnerTest.java

```diff
@@ -313,9 +313,11 @@ protected Iterator<Object[]> provideTestSqlWithExecutionException() {
     });
 
     // Timeout exception should occur with this option:
+    // - If the query times out during submission, we get: "Error occurred during stage submission: Timeout"
+    // - If the query times out during execution, we get: "Timed out on stage 0 waiting for data sent by a child stage"
     testCases.add(new Object[]{
         "SET timeoutMs = 1; SELECT * FROM a JOIN b ON a.col1 = b.col1 JOIN c ON a.col1 = c.col1",
-        "Timeout"
+        "Time"
```

Comment:
Isn't this too relaxed? It is fine to merge this PR to fix the flaky tests, but we would probably need to change the code to support regex so we can verify that it matches one of the two messages.